common/source/docs/common-esc-telem-based-notch.rst

```diff
@@ -37,7 +37,8 @@ Slewing ensures that the step changes at each update tick are smooth, but for op
 
 Checking Harmonic Notch Effectiveness
 =====================================
-Once the notch filter(s) are setup, the effectiveness of them can be checked by again measuring the  frequency spectrum of the output of the filters (which are the new inputs to the IMU sensors). Refer back to the :ref:`common-imu-batchsampling`  page for this.
+Once the notch filter(s) are setup, the effectiveness of them can be checked by again measuring the  frequency spectrum of the output of the filters (which are the new inputs to the IMU sensors). Refer back to the :ref:`common-imu-batchsampling`  or :ref:`common-raw-imu-logging` page for this.
```

Comment:
I know you just changed the links here not the text, but I don't know "which are the new inputs to the IMU sensors" means, I would just delete the bit in brackets.

Reply:
done